compiler/rustc_resolve/src/macros.rs

```diff
@@ -35,6 +37,7 @@ use rustc_span::edition::Edition;
 use rustc_span::hygiene::{self, ExpnData, ExpnKind, LocalExpnId};
 use rustc_span::hygiene::{AstPass, MacroKind};
 use rustc_span::symbol::{kw, sym, Ident, Symbol};
+use rustc_span::BytePos;
 use rustc_span::{Span, DUMMY_SP};
```

Comment:
```suggestion
use rustc_span::{BytePos, Span, DUMMY_SP};
```
Another import auto-insert.